diff --git a/setup.py b/setup.py
index 72f7a0b..b893730 100644
--- a/setup.py
+++ b/setup.py
@@ -47,13 +47,13 @@ setup(
         'spleeter.utils',
     ],
     package_data={'spleeter.resources': ['*.json']},
-    python_requires='>=3.6, <3.8',
+    python_requires='>=3.6, <3.9',
     include_package_data=True,
     install_requires=[
         'ffmpeg-python',
         'importlib_resources ; python_version<"3.7"',
         'norbert==0.2.1',
-        'pandas==0.25.1',
+        'pandas',
         'requests',
         'setuptools>=41.0.0',
         'librosa==0.7.2',
